src/App.svelte

```diff
@@ -14,6 +14,8 @@
     return parts[1] + "." + parts[0];
   };
 
+  const getAnchor = url => (url.split("#").length > 1 ? `#${url.split("#").pop()}` : "");
```

Comment:
![image](https://user-images.githubusercontent.com/503185/122184869-3b115880-cebf-11eb-8aa7-3b01418ffd26.png)

url.split.pop is enough no ?

Reply:
In the event there is no anchor:
```javascript
"http://example.com".split("#").pop() // returns: "http://example.com"
```